cmake/projects/libdill/hunter.cmake

```diff
@@ -0,0 +1,20 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME libdill
+    VERSION "1.6"
+    URL "https://github.com/isaachier/libdill/archive/hunter-1.6-p1.tar.gz"
```

Comment:
https://github.com/hunter-packages/libdill should be used?

Reply:
My bad. Fixed it.